manager/secret_manager.go

```diff
@@ -8,6 +8,30 @@ func (r *ResourceManager) CreateSecret(namespace string, secret *model.Secret) (
 	return r.kubeClient.CreateSecret(namespace, secret)
 }
 
-func (r *ResourceManager) GetSecret(namespace, name string) (secret *model.Secret, err error) {
-	return r.kubeClient.GetSecret(namespace, name)
+func (r *ResourceManager) SecretExists(namespace string, secretName string) (exists bool, err error) {
+	return r.kubeClient.SecretExists(namespace, secretName)
+}
+
+func (r *ResourceManager) GetSecret(namespace string, secret *model.Secret) (secretRes *model.Secret, err error) {
```

Comment:
Shouldn't this just take name? 
```
func (r *ResourceManager) GetSecret(namespace, name string) (secret *model.Secret, err error) {
```

Reply:
How does the API know what namespace to use?

Reply:
You'd still pass in the `namespace`,  meant shouldn't it take `name` instead of `*model.Secret`? Like this:
```
func (r *ResourceManager) GetSecret(namespace, name string) (secret *model.Secret, err error)
```

Reply:
I think I was following the CreateSecret for refactoring.. It seems like CreateSecret converts to a model inside the `secret_server.go`, instead of the `secret_manager.go`.
To verify, I should be creating the model inside the `secret_manager.go`?
And refactor CreateSecret inside of secret_server.go?